wikimedia/wikipedia-ios
# T375566 : Wikiwrapped pt 1

**Phabricator:** 
https://phabricator.wikimedia.org/T375566

### Notes
* To have entry point, test in build 170 OR uncomment lines 178-199 in WMFProfileViewModel - this PR has the entry point commented out

### Test Steps
1. Open app (read note above)
2. Navigate to profile
3. Navigate to Year in Review
4. Ensure it shows, you can navigate through the slides, press the done button, or the next button

### Screenshots/Videos

https://github.com/user-attachments/assets/ce7dcd4e-8cd0-4240-9c2d-1b92f55d1875

![Simulator Screenshot - iPhone 15 Pro - 2024-10-01 at 12 59 25](https://github.com/user-attachments/assets/18fa6fd6-bcb0-4742-abef-4bf7fd1aae09)
![Simulator Screenshot - iPhone 15 Pro - 2024-10-01 at 15 50 09](https://github.com/user-attachments/assets/da7e897c-3a05-45f0-8205-1ca00c402ee7)
![Simulator Screenshot - iPhone 15 Pro - 2024-10-01 at 15 50 24](https://github.com/user-attachments/assets/02384a32-105b-4cbc-b68c-10e1f58ad1fc)
![Simulator Screenshot - iPhone 15 Pro - 2024-10-01 at 15 50 29](https://github.com/user-attachments/assets/437be20a-f5de-4259-ae43-a022308c7d2e)
